torchvision/csrc/io/decoder/__init__.py

```diff
@@ -0,0 +1,208 @@
+import torch
```

Comment:
I thought `csrc` directory is for C++ code, why are you adding `__init__.py` here?